src/experimental-bots/filler-common/utils.ts

```diff
@@ -370,6 +376,47 @@ export const getDriftClientFromArgs = ({
 	return driftClient;
 };
 
+export const getUserFeeTier = (
```

Comment:
is this not in sdk?

Reply:
it's on the sdk but on a User object which we dont already create -- so i just copied it out so we wouldnt have to make a User using the OneShotSubscriber and an extra subscribe just to access this method